imod/msw/sprinkling.py

```diff
@@ -94,3 +94,6 @@ def _render(self, file, index, svat):
         self._check_range(dataframe)
 
         return self.write_dataframe_fixed_width(file, dataframe)
+
+    def is_regridding_supported(self) -> bool:
+        return False
```

Comment:
Please add a comment that this can be changed when this is resolved https://github.com/Deltares/imod-python/issues/728

Reply:
done